ci/deps/travis-37-cov.yaml

```diff
@@ -1,6 +1,5 @@
 name: pandas-dev
 channels:
-  - defaults
```

Comment:
The travis-37-cov build would hang here. Not sure why, but remove the defaults channel fixed it.